frontend/src/components/commons/PlaceForm.js

```diff
@@ -0,0 +1,363 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Tooltip from '@material-ui/core/Tooltip';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import CancelOutlinedIcon from '@material-ui/icons/CancelOutlined';
+import SaveOutlinedIcon from '@material-ui/icons/SaveOutlined';
+import MenuItem from '@material-ui/core/MenuItem';
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(3),
+  },
+  button: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+const initialState = {
+  primaryPictureUrl: '',
+  type: 'cityscape',
+  title: '',
+  street: '',
+  address: '',
+  latitude: '',
+  longitude: '',
+  placeDescription: '',
+  pictureDescription: '',
+  aperture: '',
+  focalLength: '',
+  shutterSpeed: '',
+  iso: '',
+  flash: '',
+  youTubeUrl: '',
+  extraOne: '',
+  extraTwo: '',
+  particularities: '',
+};
+
+const types = [
+  {
+    value: 'cityscape',
+    label: 'Stadt',
+  },
+  {
+    value: 'harbour',
+    label: 'Hafen',
+  },
+  {
+    value: 'landscape',
+    label: 'Landschaft',
+  },
+  {
+    value: 'night',
+    label: 'Nachtaufnahme',
+  },
+];
+
+export default function PlaceForm({ onSave, place = initialState }) {
+  const classes = useStyles();
+  const history = useHistory();
+  const [placeData, setPlaceData] = useState(place);
+
+  return (
+    <>
+      {/*<UploadPicture />*/}
```

Comment:
I guess this is a TODO? :) Try to keep main clean and without comments.

You can use the `form` as the enclosing tag and get rid of the fragment.

Reply:
its basically a working component that can open a file but i did not think about a way to save that image to disc and put the url into mongodb, that´s why i put a comment around it.